bigquery_etl/newtab_merino/__init__.py

```diff
@@ -0,0 +1,107 @@
+"""Export data from BigQuery to GCS."""
+
+import json
+from datetime import datetime, timedelta, timezone
+
+import rich_click as click
+from google.cloud import storage  # type: ignore
+from google.cloud import bigquery
+
+
+@click.command()
+@click.option(
+    "--source-project",
+    required=True,
+    help="Google Cloud Project where the source table is located.",
+)
+@click.option(
+    "--source-dataset",
+    required=True,
+    help="Dataset in BigQuery where the source table is located.",
+)
+@click.option(
+    "--source-table", required=True, help="Name of the source table in BigQuery."
+)
+@click.option(
+    "--destination-bucket",
+    required=True,
+    help="Destination Google Cloud Storage Bucket.",
+)
+@click.option(
+    "--destination-prefix", required=True, help="Prefix of the bucket path in GCS."
+)
+def export_newtab_merino_extract_to_gcs(
+    source_project: str,
+    source_dataset,
+    source_table,
+    destination_bucket,
+    destination_prefix,
+):
+    """Use bigquery client to export data from BigQuery to GCS."""
+    client = bigquery.Client(source_project)
+
+    # Generate the current timestamp
+    timestamp = datetime.utcnow().strftime("%Y%m%d%H%M")
+
+    # BigQuery does not export the proper JSON format, so we use a temp file and reformat
+    temp_file = "temp.json"
```

Comment:
```suggestion
    temp_file = "temp.ndjson"
```

Reply:
i think they want .json as the extension, but will see if ndjson works

Reply:
But this is the temp file? The one that uses `ndjson`. You then pull that in and rewrite it as `json` below. Merino won't be reading this file, if I understand correctly

Reply:
yeah, i misunderstood...changing :)